### PR TITLE
Urban System Lab

### PR DESCRIPTION
**Pull request by mgraber**
#4 
+ Read input data from a local csv (this is a static flat file that got sent via email --  converted from xlsx)
+ Extract and sort relevant columns for transfer postgres
+ Export to sys to transfer to postgres
+ Data mapping and cleaning
     + Combine radio-button and text input columns where relevant
     + Check that zip codes are valid
     + Strip excess white space from concatenated options
     + Simplify/normalize some text answers
+ Version and export

### CODE REVIEW

**Comment by SPTKL** [comment]
there are all NULL records, should we remove?

**Comment by mgraber** [comment]
> there are all NULL records, should we remove?

Most recent commit removes all NULL records, as well as cases where nyc_res = 'No', since these had all other fields missing.
There are still a few records where the only non-NULL data is that a respondent lives in NYC and answered 'Yes' to social distancing. Should these be included?

**Comment by SPTKL** [comment]
also should we just put the file in axway for backup

**Comment by mgraber** [comment]
> also should we just put the file in axway for backup

I've uploaded the raw data (converted to csv) to Axway manually, since it is a one-time file.

File: recipes/usl/runner.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+source $(pwd)/bin/config.sh
+BASEDIR=$(dirname $0)
+NAME=$(basename $BASEDIR)
+VERSION=$DATE
+
+(
+    cd $BASEDIR
+
```
Comment (SPTKL) [inline recipes/usl/runner.sh:9]: can you include how to get the xlsx file from axway?
```bash
mkdir -p input && mkdir output
axway_cmd get <file>.xlsx input/urban_parks_perception.xlsx
```

**Pull request merged**